src/typings/target.types.ts

```diff
@@ -5,17 +5,8 @@ import type { Range, Selection, TextEditor } from "vscode";
 // - https://github.com/microsoft/TypeScript/issues/43869
 // - https://github.com/microsoft/TypeScript/issues/43950
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-import type ModifyIfUntypedStage from "../processTargets/modifiers/ModifyIfUntypedStage";
```

Comment:
These are here so that links get properly resolved in our api docs.  Maybe we need to add `eslint/no-unused-imports` to the ignore on the previous line?  See the links in comments leading up to this line

Reply:
Guessing this applies to anywhere you've tagged them with `// eslint-disable-next-line @typescript-eslint/no-unused-vars`? Didn't notice that before :P

Reply:
Something something, Chesterton's Fence.